### PR TITLE
IE11 temp fix

### PR DESCRIPTION
**Pull request by optikalefx**
This is temp until ember fixes a bug with the range input. See https://github.com/thefrontside/emberx-range-input/issues/8

### CODE REVIEW

**Comment by Robdel12** [comment]
I'm going to close this in favor of #11. Thanks for raising this to our attention! :)

Edit: turns out I'm wrong.

File: addon/components/x-range-input.js [hunk 1]
```diff
@@ -69,12 +69,25 @@ export default Ember.Component.extend({
...
 
   /**
+   * This is only here to fix IE browsers until https://github.com/emberjs/ember.js/issues/13255 is fixed
+   *
+   * @private
+   */
+  change() {
+    if (navigator.userAgent.indexOf('Trident/') !== -1) {
+      // needs to be called with context for some reason
+      this.get('input').call(this);
```
Comment (cowboyd) [inline addon/components/x-range-input.js:79]: `this.input()` ?

Comment (fivetanley) [inline addon/components/x-range-input.js:79]: might be a computed property, probably more safe to use `this.get`

**Comment by Robdel12** [comment]
Closing this in favor of #12. This time for real :P

**Pull request closed without merging**